themes/mongodb/src/js/componentThirdParty.js

```diff
@@ -28,15 +12,16 @@ export function setup(fastNav) {
     const branch = document.body.getAttribute('data-branch');
 
     try {
-        const sampleFactor = SAMPLE_FACTORS[project] || 0.1;
-        window.delighted.survey({
-            minTimeOnPage: 180,
-            sampleFactor: sampleFactor,
-            properties: {
-              project,
-              branch,
-            }
-        });
+        if (ENABLED_SITES_FOR_DELIGHTED.has(project)) {
+            const projectName = project === 'docs' ? 'manual' : project;
```

Comment:
@jdestefano-mongo given the [comment](https://github.com/mongodb/docs-tools/pull/484#discussion_r600529348) regarding project names, would we want to honor `Manual` as the project name that's reported on the survey for Delighted?